app/views/draws/_group_report.html.erb

```diff
@@ -1,7 +1,8 @@
 <% if groups.all? { |_s, groups| groups.empty? } %>
   <p>There are no groups yet.</p>
 <% else %>
-  <% sizes.each do |size| %>
+  <% # THIS IS THE HACKY FIX %>
```

Comment:
Found it - you need to update `app/views/draws/oversubscription.html.erb` as well to pass `@sizes` instead of `@draw_sizes`.